### PR TITLE
styles: pinned footer to the bottom of page regardless of page size

### PR DESCRIPTION
**Pull request by Kamran64**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Adjusted the CSS file to pin the footer to the bottom of the page regardless of page size.

## Related Issues
Fixes: https://github.com/appsody/website/issues/136

### CODE REVIEW

**Comment by tomleah** [comment]
Footer seems to be half hidden on short documentation
image

**Comment by Kamran64** [comment]
Fixed

File: src/styles/main.css [hunk 1]
```diff
@@ -353,3 +353,8 @@ footer a:hover {
 #docs-sidebar .active{
   color: var( --colour-scheme-1);
 }
+
+.doc-content {
+  position: relative;
```
Comment (tomleah) [inline src/styles/main.css]: Not sure if `position: relative;` is needed here

Comment (Kamran64) [reply src/styles/main.css]: Tested, makes no difference. I've removed it.

**Pull request merged**